### PR TITLE
Correctly handle whitespace character(s) on filter blocks

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
"Funny" bug I encountered: the whitespace characters are not handled correctly.

### CODE REVIEW

File: askama_parser/src/node.rs [hunk 1]
```diff
@@ -577,7 +577,7 @@ impl<'a> FilterBlock<'a> {
                 ws(identifier),
                 opt(|i| Expr::arguments(i, s.level.get(), false)),
                 many0(|i| filter(i, s.level.get())),
-                opt(Whitespace::parse),
+                ws(opt(Whitespace::parse)),
```
Comment (Kijewski) [inline askama_parser/src/node.rs:580]: Wouldn't this allow `{% filter lower|indent(2) - %}`, too?

Comment (GuillaumeGomez) [reply askama_parser/src/node.rs:580]: Good catch! I'll add a (ui) test to ensure it doesn't happen by error.

**Comment by GuillaumeGomez** [comment]
Fixed and added a ui test for that.

File: testing/tests/ui/filter_block_ws.stderr [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  = note: this error originates in the derive macro `Template` (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error[E0422]: cannot find struct, variant or union type `G` in this scope
+  --> tests/ui/filter_block_ws.rs:11:5
+   |
+8  | struct A;
+   | --------- similarly named struct `A` defined here
+...
+11 |     G { v: "pIKA" }.render().unwrap();
+   |     ^ help: a struct with a similar name exists: `A`
```
Comment (djc) [inline testing/tests/ui/filter_block_ws.stderr:17]: This error is just a typo, right?

**Review by djc — APPROVED** [review]

File: testing/tests/ui/filter_block_ws.stderr (hunk 2 above)
Comment (GuillaumeGomez) [reply testing/tests/ui/filter_block_ws.stderr:17]: Oof indeed.

**Comment by GuillaumeGomez** [comment]
Fixed the test.

**Comment by GuillaumeGomez** [comment]
Didn't realize you approved it a while ago. ^^'

Merging!

**Pull request merged**